### PR TITLE
Fix linting errors

### PR DESCRIPTION
**Pull request by jessealama**
There are still some linting errors due to the new use of ``<sub>𝔻</sub>``. I would like to fix those in a separate PR, because it would require us to "fix" ecmarkup.

### CODE REVIEW

File: intl.emu [hunk 1]
```diff
@@ -85,16 +85,17 @@ location: https://github.com/tc39/proposal-decimal/
           1. Let _sign_ be ~negative~.
           1. Set _x_ to 0.
         1. <ins>If _x_ is a Decimal128 object, then</ins>
-          1. <ins>Perform ? RequireInternalSlot(_O_, [[Decimal128Data]]).</ins>
```
Comment (ptomato) [inline intl.emu:88]: I think you still need this `?`.

File: intl.emu [hunk 2]
```diff
@@ -85,16 +85,17 @@ location: https://github.com/tc39/proposal-decimal/
...
           1. <ins>Assert: _d_ is not *+∞*<sub>𝔻</sub>.</ins>
           1. <ins>Assert: _d_ is not *-∞*<sub>𝔻</sub>.</ins>
           1. <ins>Let _v_ be _d_[0].</ins>
           1. <ins>If _v_ is *-0*<sub>𝔻</sub>, then
             1. <ins>Let _sign_ be ~negative~.</ins>
             1. <ins>Set _x_ to 0.</ins>
-          1. <ins>Else if _v_ < 0, then let _sign_ be ~negative~.</ins>
+          1. <ins>Else,</ins>
+            1. <ins>If _v_ < 0, let _sign_ be ~negative~.</ins>
           1. Else, let _sign_ be ~positive~.</ins>
```
Comment (ptomato) [inline intl.emu:99]: This step probably now needs to be indented.

File: intl.emu [hunk 3]
```diff
@@ -230,19 +231,19 @@ location: https://github.com/tc39/proposal-decimal/
...
+          1. [declared="n1,r1"] Let _n1_ be an integer and _r1_ a mathematical value, with <emu-eqn>_r1_ = ToRawFixedFn(_n1_, _f_)</emu-eqn>, such that <emu-eqn>_n1_ modulo _roundingIncrement_ = 0</emu-eqn>, <emu-eqn>_r1_ ≤ _x_</emu-eqn>, and _r1_ is maximized.
           1. [declared="n2,r2"] Let _n2_ be an integer and _r2_ a mathematical value, with <emu-eqn>_r2_ = ToRawFixedFn(_n2_, _f_)</emu-eqn>, such that <emu-eqn>_n2_ modulo _roundingIncrement_ = 0</emu-eqn>, <emu-eqn>_r2_ ≥ _x_</emu-eqn>, and _r2_ is minimized.
           1. Let _r_ be ApplyUnsignedRoundingMode(_x_, _r1_, _r2_, _unsignedRoundingMode_).
           1. If _r_ is _r1_, then
-            1. Let _n_ be _n1_.
+            1. <ins>Set</ins> _n_ <del>be</del><ins>to</ins> _n1_.
             1. Let _xFinal_ be _r1_.
           1. Else,
-            1. Let _n_ be _n2_.
+            1. <del>Let</del><ins>Set</ins> _n_ <del>be</del><ins>to</ins> _n2_.
```
Comment (ptomato) [inline intl.emu:246]: I'd suggest trying to get these fixes that are not specific to Decimal, already into the ECMA-402 text. For now, it's probably clearer to just edit it to be "Set _n_ to _n1_" etc., without the `<ins>` and `<del>` tags. Otherwise it's confusing to the reader why this change is part of Decimal.

**Review by ptomato — APPROVED** [review]

File: intl.emu (hunk 1 above)
Comment (jessealama) [reply intl.emu:88]: I was going to write a long comment here but deleted it I realized, while trying to respond, there there's a mistake of mine one level higher than where it appears to be: the argument to the AO under discussion should be a Decimal128 *value*, not a Decimal128 *object*. The line can be deleted.

File: intl.emu (hunk 2 above)
Comment (jessealama) [reply intl.emu]: Ah, good spot. Fixed.

File: intl.emu (hunk 3 above)
Comment (jessealama) [reply intl.emu]: Sure, I can do that. I've simplified the change to just do "Set ...", as suggested.
